### PR TITLE
feat: add lazyStart option

### PR DESCRIPTION
**Pull request by killagu**

### CODE REVIEW

File: lib/ready.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ class Ready extends EventEmitter {
    * @param  {Object} opt
    *   - {Number} [timeout=10000] - emit `ready_timeout` when it doesn't finish but reach the timeout
    *   - {Boolean} [isWeakDep=false] - whether it's a weak dependency
+   *   - {Boolean} [lazyStart=false] - manual check if cache size is 0
```
Comment (fengmk2) [inline lib/ready.js]: comment is wrong.

**Comment by fengmk2** [comment]
ci fail

**Review by popomore — APPROVED** [review]

**Comment by killagu** [comment]
@fengmk2 CI passed and comment has modified to `will not check cache size automatically, if lazyStart is true`.

**Comment by popomore** [comment]
+ ready-callback@2.1.0

**Pull request merged**